en/docs/get-started/about-this-release.md

```diff
@@ -56,8 +56,8 @@ For more information on WSO2 API Manager, see the [overview]({{base_path}}/getti
 ??? note "Support for managing Rate Limiting Policies with APICTL"
     This new capability allows you to import and export rate limiting policies between environments. Rate limiting policies are imported to the target environment in a separate stage before importing either the API or the App. After importing rate limiting policies to the target environment, you may import the API/Application without encountering any rate limiting policy support problems.
 
-??? note "MI Dashboard Revamp and Pagination and Search capability for MI Management API"
-    This feature introduces a new Micro Integrator Control plane and Implement Pagination and Search Functionality.
+??? note "MI Dashboard Revamp and Search capability for MI Management API"
+    This feature revamps the MI Dashboard and Implements Search Functionality for MI Management API.
```

Comment:
Is it really a revamp? It feels like we are overselling this. I feel like this is just enabling search for artifacts, etc.; in one instance, a nested search is possible. 

The only revamp done is the removal of the database, but I'm not sure we can call it a revamp since this doesn't seem to impact the user much.

Shall we say something like this:

```suggestion
    This feature provides some improvements to the Micro Integrator Dashboard and adds a new search functionality to the Micro Integrator Management API.
```

Reply:
Shall we say **This feature improves the MI Dashboard and Implements Search Functionality for MI Management API**.?